subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -914,7 +930,28 @@ static void isr_rx_done_cleanup(struct lll_sync *lll, uint8_t crc_ok, bool sync_
 
 static void isr_done(void *param)
 {
+	struct lll_sync *lll;
+
 	lll_isr_status_reset();
+
+	lll = param;
+	if (lll->is_aux_sched) {
+		struct node_rx_pdu *node_rx;
+
+		lll->is_aux_sched = 0U;
+
+		node_rx = ull_pdu_rx_alloc();
+		LL_ASSERT(node_rx);
+
+		node_rx->hdr.type = NODE_RX_TYPE_EXT_AUX_RELEASE;
+
+		node_rx->hdr.rx_ftr.param = lll;
+		node_rx->hdr.rx_ftr.aux_failed = 1U;
```

Comment:
Not sure I understand why we are marking it as failed here. Can you elaborate and/or in-code comment?

Reply:
Added comment in code.